code/src/App.js

```diff
@@ -1,9 +1,47 @@
 import React from 'react'
+import {Provider} from 'react-redux'
+import { combineReducers, createStore } from '@reduxjs/toolkit'
+
+import { todos } from 'reducers/todos'
+import { AddTodo } from 'components/AddTodo'
+import { TodoList } from 'components/TodoList'
+
+import {
+  Wrapper,
+  ProviderContainer
+} from '../src/components/Styling'
```

Comment:
Nice way of organizing your styled components. I still haven't find any structure for it so I keep adding all of them in several components. Will try this next time I think! 💯